web/src/components/Settings/DataStorePlugin/forms/OpenSearch/OpenSearch.tsx

```diff
@@ -1,5 +1,31 @@
+import {Form, Input} from 'antd';
+import * as S from '../../DataStorePluginForm.styled';
+import AddressesList from './AddressesList';
+
 const OpenSearch = () => {
-  return <div>OpenSearch</div>;
+  return (
+    <S.FormContainer>
+      <S.FormColumn>
+        <Form.Item label="Username" name="username">
```

Comment:
The names from this form does not match the `TDraftConfig` structure. This is going to create a different object structure than the expected in the config Services.